src/components/WorkOrder/EditWorkOrder.tsx

```diff
@@ -39,14 +50,48 @@ const EditWorkOrder = ({ workOrderReference }: EditWorkOrderProps) => {
   const router = useRouter()
 
   useEffect(() => {
-    fetchWorkOrder()
+    fetchWorkOrderDetails()
   }, [workOrderReference])
 
-  const fetchWorkOrder = async () => {
+  const fetchWorkOrderDetails = async () => {
     setLoading(true)
-    const { response, error } = await getWorkOrder(workOrderReference)
-    setWorkOrder(response)
-    setError(error)
+    const workOrderResponse = await getWorkOrder(workOrderReference)
+
+    if (!workOrderResponse.success) {
+      setError(workOrderResponse.error.message)
```

Comment:
Forgot to edit the model?

https://github.com/LBHackney-IT/repairs-hub-frontend/blob/c3cf73b7e2584e1d487e4d8ec3e9dd2d55c28e34/src/types/requests/types.ts#L1-L5